mod/system_log/languages/de.php

```diff
@@ -19,7 +19,7 @@
 	'logbrowser:user:name' => 'Benutzer',
 	'logbrowser:user:guid' => 'Benutzer-GUID',
 	'logbrowser:object' => 'Objekt-Typ',
-	'logbrowser:object:guid' => 'Objekt-GUID',
+	'logbrowser:object:id' => 'Objekt-GUID',
```

Comment:
remove the old translation in the non-English translations or leave it. A language pull will remove the non translated (on Transifex) next time anyway

Reply:
not a problem for now... this way the 'master' installation has correct translations/keys